### PR TITLE
Quick fixes from content review

### PR DESCRIPTION
**Pull request by agraebe**
fixing low hanging fruits from #769. included almost all "structural changes" to the docs project, so that folks can focus on content editing

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -285,7 +285,7 @@ async function redirects() {
     },
     {
       source: '/core/smart/rpc-api.html',
-      destination: '/references/stacks-rpc-api',
+      destination: '/references/stacks-blockchain',
```
Comment (aulneau) [inline next.config.js:288]: shouldn't this be `stacks-blockchain-api`?

Comment (agraebe) [reply next.config.js:288]: yeah, good call. i didn't review the name but just copy/pasted it. let me rename and add a new redirect

**Review by hstove — APPROVED** [review]
Looks good! Appreciate the architecture changes relating to indexing/radiks.

Looks like there is a build failure in your latest commit. I haven't ran locally to debug the issue. The content looks good, though.

**Comment by agraebe** [comment]
> Looks good! Appreciate the architecture changes relating to indexing/radiks.
> 
> Looks like there is a build failure in your latest commit. I haven't ran locally to debug the issue. The content looks good, though.

thanks! fixing now

**Pull request merged**